src/views/TimersView.tsx

```diff
@@ -0,0 +1,80 @@
+import { useTimers } from "../context/TimerContext";
+import Stopwatch from "../components/timers/Stopwatch";
+import Countdown from "../components/timers/Countdown";
+import XY from "../components/timers/XY";
+import Tabata from "../components/timers/Tabata";
+import type { Timer } from "../types/types";
+import { TimerContainer } from '../components/generic/ContainerDisplays';
+import Button from '../components/generic/Button';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPause, faPlay } from "@fortawesome/free-solid-svg-icons";
+import { TimerStyle } from "../components/generic/FormStyling";
+import { ButtonContainer } from '../components/generic/ContainerDisplays'
+
+const TimersView = () => {
+  const {
+    timers,
+    currentTimerIndex,
+    isWorkoutRunning,
+    startWorkout,
+    pauseWorkout,
+    resetWorkout,
+    fastForward,
+  } = useTimers();
+
+  const renderTimer = (timer: Timer) => {
+    switch (timer.type) {
+      case "stopwatch":
+        return <Stopwatch id={timer.id} />;
+      case "countdown":
+        return <Countdown id={timer.id} />;
+      case "xy":
+        return <XY id={timer.id} />;
+      case "tabata":
+        return <Tabata id={timer.id} />;
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <>
+      <h2>Timers</h2>
+      <ButtonContainer>
```

Comment:
Disable the buttons when they should not be able to be clicked (ex. You should not be able to press Play when the workout has come to an end, but before you have pressed reset). Otherwise, users can (and will) click them and get into unexpected states/bugs.